src/test/server/postgres_wire_handler_test.cpp

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Do you see any possibility to increase your test coverage?

Reply:
... we have since added tests to validate the behavior of the ServerSession, using Mock objects for the client connection and the task runner (these two classes are difficult to test because they're basically wrappers around boost's io_service and tcp socket).

Although the overall coverage number has still decreased, we attempted to cover all important parts and leave the untested/untestable code as simple as possible.